xslt/auto.xslt

```diff
@@ -133,19 +134,36 @@
             </xsl:if>
             <fo:table-cell xsl:use-attribute-sets="td">
                 <fo:block>
-                    <!-- attach id to first finding of each threatLevel so pie charts can link to it -->
-                    <xsl:if test="@id">
-                        <xsl:attribute name="id">
-                            <xsl:value-of select="@id"/>
-                        </xsl:attribute>
-                    </xsl:if>
-                    <fo:basic-link xsl:use-attribute-sets="link">
-                        <xsl:attribute name="internal-destination">
-                            <xsl:value-of select="@findingId"/>
-                        </xsl:attribute>
-                        <xsl:value-of select="findingNumber"/>
-                    </fo:basic-link>
+                    <fo:inline>
+                        <!-- attach id to first finding of each threatLevel so pie charts can link to it -->
+                        <xsl:if test="@id">
+                            <xsl:attribute name="id">
+                                <xsl:value-of select="@id"/>
+                            </xsl:attribute>
+                        </xsl:if>
+                        <fo:basic-link xsl:use-attribute-sets="link">
+                            <xsl:attribute name="internal-destination">
+                                <xsl:value-of select="@findingId"/>
+                            </xsl:attribute>
+                            <xsl:value-of select="findingNumber"/>
+                        </fo:basic-link>
+                    </fo:inline>
                 </fo:block>
+                <!-- <fo:block>
```

Comment:
Would you mind removing remarked code? Apologies for nit-picking, but that keeps the codebase cleaner